djaoapp/templates/jinja2/saas/profile/_form.html

```diff
@@ -21,6 +21,50 @@
 {{ phone_input_field(form['phone'], phone_verified_at, form.hide_labels) }}
 {% endif %}
 
+{% if form['street_address'] %}
+    <label class="col-form-label requiredField">Street address</label>
```

Comment:
This code should be moved into a `street_address_input_field` macro in 'djaoapp/templates/jinja2/_form_fields.html'.